manifests/charts/gateways/istio-ingress/templates/deployment.yaml

```diff
@@ -74,9 +77,18 @@ spec:
             - -c
             - sysctl -w kernel.core_pattern=/var/lib/istio/data/core.proxy && ulimit -c unlimited
           securityContext:
+          {{- if not $gateway.runAsRoot }}
```

Comment:
This should be reverted - `enableCoreDump` requires root. It does not make sense to ever have this non root as it will always fail.